pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -575,7 +575,7 @@ func (rc *reconciler) updateStates(volumesNeedUpdate map[api.UniqueVolumeName]*r
 			continue
 		}
 		if volume.pluginIsAttachable {
-			err = rc.actualStateOfWorld.MarkDeviceAsMounted(volume.volumeName)
+			err = rc.actualStateOfWorld.MarkDeviceAsMounted(volume.volumeName, volume.devicePath)
```

Comment:
Yes, this is the place I am still fuzzy. 

@saad-ali @jingxu97 How often is `updateStates` called? Since `WaitForAttach` queries cloudprovider API, I am concerned if calling `WaitForAttach` in every `updateStates`, we add too much API traffic.